### ISSUE TITLE
scyll_setup fail: This is an unsupported kernel version: kernel-3.10.0-862.2.3.el7.x86_64

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#3439]
*Installation details*
Scylla version (or git commit hash): 666.development-0.20180515.83e64192d
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS7

Artifact-test job link: [job/scylla-master-centos-rpm-test/406](http://jenkins.cloudius-systems.com:8080/job/scylla-master-centos-rpm-test/406/label=muninn/artifact/latest/job.log)

I'm using the latest centos7, and the kernel is updated to latest (3.10.0-862.2.3.el7.x86_64)

```
[scylla-test@amos-centos7-test2 ~]$ sudo yum update
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.trouble-free.net
 * epel: reflector.westga.edu
 * extras: reflector.westga.edu
 * updates: ftp.linux.ncsu.edu
No packages marked for update
[scylla-test@amos-centos7-test2 ~]$

[scylla-test@amos-centos7-test2 ~]$ sudo scylla_setup
Skip any of the following steps by answering 'no'
Do you want to run kernel version check?
Answer yes to have this script verify that the currently installed kernel is qualified to run Scylla; answer no to skip this check.
[YES/no]YES
This is an unsupported kernel version.
Please upgrade to a newer kernel version.
 see http://www.scylladb.com/kb/kb-fs-not-qualified-aio/ for details
[scylla-test@amos-centos7-test2 ~]$ echo $?
1

[scylla-test@amos-centos7-test2 ~]$ uname -a
Linux amos-centos7-test2 3.10.0-862.2.3.el7.x86_64 #1 SMP Wed May 9 18:05:47 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by avikivity** [comment]
It's scylladb/seastar#448 and scylladb/seastar#449.

**Comment by amoskong** [comment]
Thanks.

```
[scylla-test@amos-centos7-test2 ~]$ mkdir -p /var/tmp/mnt
[scylla-test@amos-centos7-test2 ~]$ dd if=/dev/zero of=/var/tmp/kernel-check.img bs=1M count=128
128+0 records in
128+0 records out
134217728 bytes (134 MB) copied, 0.0741923 s, 1.8 GB/s
[scylla-test@amos-centos7-test2 ~]$ mkfs.xfs /var/tmp/kernel-check.img
meta-data=/var/tmp/kernel-check.img isize=512    agcount=4, agsize=8192 blks
         =                       sectsz=512   attr=2, projid32bit=1
         =                       crc=1        finobt=0, sparse=0
data     =                       bsize=4096   blocks=32768, imaxpct=25
         =                       sunit=0      swidth=0 blks
naming   =version 2              bsize=4096   ascii-ci=0 ftype=1
log      =internal log           bsize=4096   blocks=855, version=2
         =                       sectsz=512   sunit=0 blks, lazy-count=1
realtime =none                   extsz=4096   blocks=0, rtextents=0
[scylla-test@amos-centos7-test2 ~]$ mount /var/tmp/kernel-check.img /var/tmp/mnt -o loop
mount: only root can use "--options" option
[scylla-test@amos-centos7-test2 ~]$ sudo mount /var/tmp/kernel-check.img /var/tmp/mnt -o loop
[scylla-test@amos-centos7-test2 ~]$ df
Filesystem     1K-blocks    Used Available Use% Mounted on
/dev/sda1       10474496 2027568   8446928  20% /
devtmpfs         1803812       0   1803812   0% /dev
tmpfs            1811564       0   1811564   0% /dev/shm
tmpfs            1811564    9576   1801988   1% /run
tmpfs            1811564       0   1811564   0% /sys/fs/cgroup
tmpfs             362316       0    362316   0% /run/user/1025
/dev/loop0        127652    6728    120924   6% /var/tmp/mnt
[scylla-test@amos-centos7-test2 ~]$ iotune --fs-check --evaluation-directory /var/tmp/mnt
WARN  2018-05-17 08:41:42,002 [shard 0] seastar - Unable to set SCHED_FIFO scheduling policy for timer thread; latency impact possible. Try adding CAP_SYS_NICE
ERROR 2018-05-17 08:41:42,005 [shard 0] seastar - Exiting on unhandled exception: std::system_error (error system:13, Permission denied)
[scylla-test@amos-centos7-test2 ~]$ sudo iotune --fs-check --evaluation-directory /var/tmp/mnt
Reactor stalled for 0 ms on shard 0.
Backtrace:
  0x00000000004cbd7c
  0x00000000004cc236
  /lib64/libpthread.so.0+0x000000000000f6cf
  /lib64/libc.so.6+0x00000000000f8ec8
  0x0000000000691738
  0x00000000006911cc
  0x000000000042cbeb
  0x000000000042f71a
  0x000000000041fb2d
ERROR 2018-05-17 08:41:52,569 [shard 0] seastar - Exiting on unhandled exception: std::out_of_range (sstring::substr out of range)
[scylla-test@amos-centos7-test2 ~]$ echo $?
1
```

**Comment by glommer** [comment]
Can I get access ? I can repro none of that locally.

**Comment by avikivity** [comment]
Will be fixed by 98967da94f24597eeac8be9038c6d0974563bc14.
